File: Java/armstrong.java
import java.util.*;
public class armstrong{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number:");
        int a = sc.nextInt();
        sc.close();
        int num=a,sum=0;
        while(num>0){
            int d = num%10;
            sum+=d*d*d;
            num = num/10;   
        }
        if(a==sum){
            System.out.println("Armstrong");
        }
        else{
            System.out.println("Not Armstrong");
        }
    }
}

File: Python/armstrong.py
import math
def is_armstrong(num):
    nod = int(math.log10(num)+1)
    n = num
    sum=0
    while(num>0):
        d = num%10
        sum = sum + math.pow(d,nod)
        num = num //10
    return sum == n
def main():
    n = int(input("Enter a no:"))
    if(is_armstrong(n)):
        print("Armstrong No")
    else:
        print("Not Armstrong No")
if __name__ == "__main__":
    main()